### PR TITLE
fix: Hide empty bar under toolbar

### PR DESCRIPTION
**Pull request by acezard**

### CODE REVIEW

File: src/styles/index.css [hunk 1]
```diff
@@ -131,6 +131,11 @@ html .akEditor > div:first-child > div:first-child + div hr {
     height: calc(100%);
 }
 
+/* Hacky CSS to hide anything that isn't the toolbar in the atlaskit toolbar wrapper (avatar component for instance) */
+[data-testid="ak-editor-main-toolbar"] > :not(:first-child) {
+    display: none !important;
+}
+
```
Comment (Crash--) [inline src/styles/index.css:138]: I'm not confortable with this fix. 

Just below we already have CSS that hides avatar. Can't we fix it? 
https://github.com/cozy/cozy-notes/blob/aa444ff6c169a8c3c311cb508f917bb39a88bbd5/src/styles/index.css#L139-L146

Comment (acezard) [reply src/styles/index.css:138]: @Crash-- no, the fixme you're linking only hide the avatar, but not their wrapper. I'm hiding their wrapper with a wildcard (heavy handed sure, but there should only be the editor toolbar in there)

**Review by Crash-- — APPROVED** [review]
🤷‍♂️

**Comment by acezard** [comment]
remove old fixme

**Pull request merged**